qtaf-xray-plugin/src/main/java/de/qytera/qtaf/xray/config/XrayConfigHelper.java

```diff
@@ -63,6 +63,18 @@ public class XrayConfigHelper {
      * The selector for accessing the configured assignee to assign text execution issues to.
      */
     public static final String RESULTS_UPLOAD_ASSIGNEE = "xray.resultsUpload.assignee";
+    /**
+     * The selector for accessing the configured assignee to assign text execution issues to.
+     */
+    public static final String RESULTS_UPLOAD_ENVIRONMENTS_ENABLED = "xray.resultsUpload.environments.enabled";
+    /**
+     * The selector for accessing the configured assignee to assign text execution issues to.
+     */
+    public static final String RESULTS_UPLOAD_ENVIRONMENTS_OS = "xray.resultsUpload.environments.os";
+    /**
+     * The selector for accessing the configured assignee to assign text execution issues to.
+     */
+    public static final String RESULTS_UPLOAD_ENVIRONMENTS_DRIVER = "xray.resultsUpload.environments.driver";
```

Comment:
The Javadoc comments do not match the configuration keys, I think.

Reply:
Ok, fixed it